doozer/doozerlib/image.py

```diff
@@ -459,11 +460,25 @@ def covscan(self, cc: coverity.CoverityContext) -> bool:
                 return False
 
     def calculate_config_digest(self, group_config, streams):
-        image_config: Dict[str, Any] = self.config.primitive()  # primitive() should create a shallow clone for the underlying dict
+        ignore_keys = ["owners", "scan_sources", "content.source.ci_alignment"]  # list of keys that shouldn't be involved in config digest calculation
```

Comment:
Do you think we can also ignore `content.source.git` fields? If there's a change in repo or Dockerfile path, scan-sources should notice in advance that SHAs do not match latest builds, and mark the component as changed.

 This could avoid mass rebuilds with PRs like [4.15 ocp-build-data moved to openshift-eng](https://github.com/openshift-eng/ocp-build-data/pull/3643)